### DISCUSSION TITLE
Possible to avoid connection failure due to wrong date/time synchronization [on client side]

### DISCUSSION THREAD

**Discussion by acrolink** [strongswan/strongswan#886] [Q&A] [answered]
Client is `Android `systems attempting to connect to `IKEv2 `server. Works fine except if the date/time on clients is wrong [out of sync with internet time(real time)]. If the date is way back or simply wrong, authentication and connection with the `StrongSwan `server fails. My question, is it possible to make it work even if the date and time on client are wrong? Thank you.

**Comment by tobiasbrunner** [comment, marked as answer]
No, checking the lifetimes of certificates can't generally be disabled. There is a [plugin that can handle invalid system time at startup](https://wiki.strongswan.org/projects/strongswan/wiki/SystimeFixPlugin), but it's intended for controlled environments where the initial system time is clearly defined (e.g. as 1970-01-01).
